web/src/LogPane.tsx

```diff
@@ -43,6 +44,7 @@ let LogLinePrefixRoot = styled.span`
   box-sizing: border-box;
   display: inline-block;
   background-color: ${color.grayDark};
+  border-top: 1px dotted transparent;
   border-right: 1px dotted ${color.grayLightest};
```

Comment:
I was playing with this on my machine and think it's clearer if the vertical line separating the prefixes from logs is more prominent than the horizontal lines separating each context.

So, the root styles change like so:
```
border-right: 1px solid ${color.grayLightest};
```
> ~~border-top: 1px dotted transparent;~~ (To be handled with negative margin, below)

...and `is-contextChange` gets these:

```
    margin-top: -1px; // Don't let border change height of element
    border-top: 1px dotted ${color.grayLightest};
```

Reply:
done!